### DISCUSSION TITLE
Using `:::` inside HTML

### DISCUSSION THREAD

**Discussion by rudolfbyker** [mkdocstrings/mkdocstrings#410] [Q&A]
I'm using mkdocstrings 0.18.1 with mkdocstrings-python 0.6.6. Is it possible to document python code inside HTML?

This works:

```
::: my_function
```

But this just prints `::: my_function` verbatim inside the collapsible summary thing:

```
<details>
<summary>Here are more technical details.</summary>

The documentation of this function contains details about what is possible in general:

::: my_function

</details>
```

**Comment by pawamoy** [comment]
Can you try this?

```md
<details markdown="1">
<summary>Here are more technical details.</summary>

The documentation of this function contains details about what is possible in general:

::: my_function

</details>
```

Also, moving this to Discussions as it's more a question about Python-Markdown and extensions rather than mkdocstrings :slightly_smiling_face:

**Comment by rudolfbyker** [comment]
I did try that (with this extension enabled https://python-markdown.github.io/extensions/md_in_html/ ). I don't have a screenshot of the results, though, but I think it did not make a difference.

**Comment by pawamoy** [comment]
You can try to use an admonition then:

```md
??? note "Here are more technical details."
    The documentation of this function contains details about what is possible in general:

    ::: my_function
```
